src/chttpd.erl

```diff
@@ -820,6 +820,10 @@ error_info({error, {illegal_database_name, Name}}) ->
     {400, <<"illegal_database_name">>, Message};
 error_info({_DocID,{illegal_docid,DocID}}) ->
     {400, <<"illegal_docid">>,DocID};
+error_info({error, {database_name_too_long, DbName}}) ->
+    {400, database_name_too_long,
```

Comment:
Is the second tuple's element is intentionally an atom? It seems to be a binary at the rest of the responses.